support/cas-server-support-shibboleth-attributes/src/main/java/org/apereo/cas/config/ShibbolethAttributeResolverConfiguration.java

```diff
@@ -77,12 +78,11 @@ public IPersonAttributeDao shibbolethPersonAttributeDao() {
                     this.applicationContext
             );
 
-            final Collection<DataConnector> connectors =
-                    Sets.newHashSet(BeanFactoryUtils.beansOfTypeIncludingAncestors(tempApplicationContext, DataConnector.class).values());
+            final Collection<DataConnector> connectors = new HashSet<>(beansOfTypeIncludingAncestors(tempApplicationContext, DataConnector.class).values());
             final AttributeResolverImpl impl = new AttributeResolverImpl();
             impl.setId(getClass().getSimpleName());
             impl.setApplicationContext(tempApplicationContext);
-            impl.setAttributeDefinitions(BeanFactoryUtils.beansOfTypeIncludingAncestors(tempApplicationContext, AttributeDefinition.class).values());
+            impl.setAttributeDefinitions(beansOfTypeIncludingAncestors(tempApplicationContext, AttributeDefinition.class).values());
```

Comment:
This something we should add to checkstyle? My preference is to always explicitly call class unless the method in question has become an idiom like `assertTrue`. But that matters not; Whether explicit or not, we probably should stick to  consistent style, or make this exception known. Right?